Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:8
+
+#copy files to the /var/www/html directory
+COPY . /var/www/html/
```

Comment:
I would prefer using another path that is not a standard web root (such as `/opt/mongodb-honeyproxy`)

Reply:
Just changed it :) @Plazmaz